.goreleaser.yml

```diff
@@ -18,18 +16,24 @@ snapshot:
   name_template: "{{ incpatch .Version }}-snapshot"
 
 dockers:
-- image_templates:
-  - "docker.io/vshn/k8up:v{{ .Version }}"
-  - "quay.io/vshn/k8up:v{{ .Version }}"
+- &image_template
+  id: docker-amd64
+  image_templates:
+  - "ghcr.io/k8up-io/k8up:v{{ .Version }}"
+  - "quay.io/k8up-io/k8up:v{{ .Version }}"
 
   # For prereleases, updating `latest` and the floating tags of the major
   # version does not make sense. Only the image for the exact version should
   # be pushed.
-  - "{{ if not .Prerelease }}docker.io/vshn/k8up:v{{ .Major }}{{ end }}"
-  - "{{ if not .Prerelease }}quay.io/vshn/k8up:v{{ .Major }}{{ end }}"
+  - "{{ if not .Prerelease }}ghcr.io/k8up-io/k8up:v{{ .Major }}{{ end }}"
+  - "{{ if not .Prerelease }}quay.io/k8up-io/k8up:v{{ .Major }}{{ end }}"
 
-  - "{{ if not .Prerelease }}docker.io/vshn/k8up:latest{{ end }}"
-  - "{{ if not .Prerelease }}quay.io/vshn/k8up:latest{{ end }}"
+  - "{{ if not .Prerelease }}ghcr.io/k8up-io/k8up:latest{{ end }}"
+  - "{{ if not .Prerelease }}quay.io/k8up-io/k8up:latest{{ end }}"
+  goarch: amd64
+- <<: *image_template
+  id: docker-arm64
+  goarch: arm64
```

Comment:
How does it work exactly with ARM images? Will there be dedicated ARM tags? I don't see any reference to docker manifests in GH actions

Reply:
I think a lot more is needed to make multiarch images work.
See:
* https://github.com/ccremer/kubernetes-zfs-provisioner/blob/master/.goreleaser.yml
* https://github.com/ccremer/kubernetes-zfs-provisioner/blob/master/.github/workflows/release.yml

I think it would make sense for a separate PR for this.

Reply:
I think a lot more is needed to make multiarch images work.

> https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md

You're right.

> I think it would make sense for a separate PR for this.

Yes, certainly.